packages/styles/stories/Components/dialog/default-dialog.example.html

```diff
@@ -1,37 +1,28 @@
 
 <section class="fd-dialog-docs-static fd-dialog fd-dialog--active">
-    <div class="fd-dialog__content" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
+    <div class="fd-dialog__content fd-dialog__content--s" role="dialog" aria-modal="true" aria-labelledby="dialog-title-1">
```

Comment:
adding `fd-dialog__content--s` is not the default case. In my opinion it should be as before. The content of the body is specified by the case. 

This is what's in the specs. Only the vertical paddings are set by default. The horizontal can be responsive, or set for all screens with modifier classes.
![Screenshot 2023-10-31 at 3 18 48 PM](https://github.com/SAP/fundamental-styles/assets/39598672/eec46bd4-2ff7-4f23-93fe-d9c4d1e360d5)